cinema_service/settings.py

```diff
@@ -134,7 +140,7 @@
 STATIC_URL = "static/"
 
 MEDIA_URL = "/media/"
-MEDIA_ROOT = BASE_DIR / "media"
+MEDIA_ROOT = "/files/media"
```

Comment:
The MEDIA_ROOT is hardcoded. It's better to construct this path using the BASE_DIR. This way, the media files will be stored in a directory relative to the project directory.